docs/content/docs/tasks/restart-application-deployment/_index.md

```diff
@@ -0,0 +1,165 @@
+---
+title: Restart an Application Deployment
+description: Learn how to restart an unsuccessful Keptn Application Deployment .
+icon: concepts
+layout: quickstart
+weight: 20
+hidechildren: true # this flag hides all sub-pages in the sidebar-multicard.html
+---
+
+## Restart an Application Deployment
+
+During the deployment of a `KeptnApp`, it might be that the deployment fails due to a unsuccessful pre-deployment evaluation or pre-deployment check.
+This might be because of, e.g., a misconfigured target value of a `KeptnEvaluationDefinition`, or of a wrong URL being checked in a pre deployment check.
+
+To retry an `KeptnApp` deployment without bumping up the version of the `KeptnApp`, we introduced the concept of **revisions** for a `KeptnAppVersion`. This means that
+whenever the spec of a `KeptnApp` changes, even though the version stays the same, the KLT Operator will create a new revision of the `KeptnAppVersion` referring to the `KeptnApp`.
+
+This way, when a `KeptnApp` failed due to a misconfigured pre deployment task, you can first fix the configuration of the `KeptnTaskDefinition`/`KeptnEvaluationDefinition`, then
+increase the value of `spec.revision` of the `KeptnApp` and finally apply the updated `KeptnApp` manifest. Afterwards, all related `KeptnWorkloadInstances` will automatically refer to the newly
+created revision of the `KeptnAppVersion` to determine whether they are allowed to enter their respective deployment phase.
```

Comment:
```suggestion
increase the value of `spec.revision` of the `KeptnApp` and finally apply the updated `KeptnApp` manifest.
This will result in a restart of the `KeptnApp`.
Afterwards, all related `KeptnWorkloadInstances` will automatically refer to the newly
created revision of the `KeptnAppVersion` to determine whether they are allowed to enter their respective deployment phase.
```